tools/automatic_query_fixer/src/main/java/com/google/cloud/bigquery/utils/queryfixer/service/BigQueryService.java

```diff
@@ -1,52 +1,69 @@
 package com.google.cloud.bigquery.utils.queryfixer.service;
 
+import com.google.api.gax.paging.Page;
 import com.google.cloud.bigquery.BigQuery;
 import com.google.cloud.bigquery.BigQueryException;
 import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.DatasetId;
 import com.google.cloud.bigquery.Job;
 import com.google.cloud.bigquery.JobInfo;
 import com.google.cloud.bigquery.QueryJobConfiguration;
+import com.google.cloud.bigquery.Table;
+
+import lombok.NonNull;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
 
 /**
  * A service to connect with the BigQuery server. It is used to communicate with the server like
  * sending queries and receiving data.
- * */
+ */
 public class BigQueryService {
 
-  private final BigQuery bigquery;
+  private static final int TableFetchSize = 1000;
```

Comment:
Its recommended to use UPPER_SNAKE_CASE for constant variable names. "TABLE_FETCH_SIZE"

Reply:
Done